mavis/annotate/variant.py

```diff
@@ -710,28 +710,34 @@ def annotate_events(
     for i, bpp in enumerate(bpps):
         log('({} of {}) gathering annotations for'.format(i + 1, total), bpp)
         bpp.data[COLUMNS.validation_id] = bpp.data.get(COLUMNS.validation_id, str(uuid()))
-        ann = _gather_annotations(
+        ann_list = _gather_annotations(
             annotations,
             bpp,
             proximity=max_proximity
         )
         for f in filters:
-            ann = f(ann)  # apply the filter
-        results.extend(ann)
-        for j, a in enumerate(ann):
-            a.data[COLUMNS.annotation_id] = '{}-a{}'.format(a.validation_id, j + 1)
+            ann_list = f(ann_list)  # apply the filter
+        results.extend(ann_list)
+        for j, ann in enumerate(ann_list):
```

Comment:
Perhaps I am just a bit unfamiliar with this, but what is `j` here? Can another name be used to clarify it?

Reply:
'j' is the index of the enumerated ann_list, like 'i' is for bpps.
The 'i', 'j', 'k', ... indexing convention is very old.  Comes from FORTRAN variable naming rules.

Reply:
Oh I understand! I'm wasn't very familiar with how enumerate() works in python for loops, so I forgot the first value is an index. Thanks for taking the time to clarify that, Dustin.